welcome/index.adoc

```diff
@@ -98,10 +98,9 @@ endif::openshift-origin[]
 
 - **xref:../installing/installing_bare_metal/installing-bare-metal.adoc#installing-bare-metal[Install a user-provisioned cluster on bare metal]**: If none of the available platform and cloud provider deployment options meet your needs, you can install {product-title} on user-provisioned, bare-metal infrastructure.
 
-- **Install a cluster on {rh-openstack-first}**: You can install a cluster on
-xref:../installing/installing_openstack/installing-openstack-installer-custom.adoc#installing-openstack-installer-custom[{rh-openstack} with customizations], with xref:../installing/installing_openstack/installing-openstack-installer-kuryr.adoc#installing-openstack-installer-kuryr[network customizations], or xref:../installing/installing_openstack/installing-openstack-installer-restricted.adoc#installing-openstack-installer-restricted[on a restricted network] on installer-provisioned infrastructure.
-+
-You can install a cluster on xref:../installing/installing_openstack/installing-openstack-user.adoc#installing-openstack-user[{rh-openstack} with customizations] or xref:../installing/installing_openstack/installing-openstack-user-kuryr.adoc#installing-openstack-user-kuryr[with network customizations] on user-provisioned infrastructure.
+- **Install a cluster on {rh-openstack-first}**: You can install a cluster on {rh-openstack} with customizations on a restricted network on installer-provisioned infrastructure. For more information, see the following documentation:
```

Comment:
Wait, I think these should be listed as distinct options. We have customizations and we have restricted networks, they can be used together, but doesn't need to.